addons/dev-helpers/dump.pl

```diff
@@ -70,14 +70,14 @@ sub _run {
     print Data::Dumper::Dumper(\%pf::config::ConfigFloatingDevices);
 }
 
-package pf::dump::vlan_filters;
+package pf::dump::access_filters;
 use base qw(pf::dump::cmd);
 use Data::Dumper;
 __PACKAGE__->mark_as_loaded();
 
 sub _run {
-    require pf::vlan::filter;
```

Comment:
Why don't you keep pf::vlan::filter ? I like the way to have that explicit and use the same logic for radius per example (pf::radius::filter)